app/models/user.rb

```diff
@@ -2,4 +2,9 @@ class User < ApplicationRecord
   devise :remote_user_authenticatable # We don't want other (default) Devise modules
   has_many :bundle_contexts
   validates :sunet_id, presence: true, uniqueness: true, null: false
+
+  def email
+    "#{sunet_id}@stanford.edu"
```

Comment:
Ah the sunet_id is already in email form:
```
 #<User id: 1, sunet_id: "saravs@stanford.edu", created_at: "2018-09-17 19:55:49", updated_at: "2018-09-17 19:55:49">
```

Reply:
removed this from the model, now just using sunet_id as the email

Reply:
I'd actually like this back.  We need to keep `sunet_id` and `email` distinct in our thinking, and the best way to do that is have a method that always returns the whole email.  For now, it would be fine to only append `unless sunet_id.include('@')`.  Then this method stays correct while we work on converting what we store.